code/lib/react-dom-shim/preset.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/preset');
```

Comment:
This doesn't have to exist if we do this: https://github.com/storybookjs/storybook/pull/21197/files#r1115437543